src/event-log/event-log-level.ts

```diff
@@ -15,101 +18,114 @@ type EventLogLevelConfig = {
   createLevelDatabase?: typeof createLevelDatabase,
 };
 
-const WATERMARKS_SUBLEVEL_NAME = 'watermarks';
 const CIDS_SUBLEVEL_NAME = 'cids';
```

Comment:
To reduce cognitive load, I think the DB name should be identified by the value it stores (watermarks) as opposed to the keys (cids).

```suggestion
const WATERMARK_SUBLEVEL_NAME = 'watermarks';
```

Also style consideration: consider just declaring this const within the method that uses it, no one else references/cares about it in this case.

Reply:
No longer relevant.